.github/CODEOWNERS

```diff
@@ -2381,6 +2381,8 @@ x-pack/test/profiling_api_integration @elastic/obs-ux-infra_services-team
 x-pack/plugins/observability_solution/observability_shared/public/components/profiling @elastic/obs-ux-infra_services-team
 
 # Shared UX
+/x-pack/test/functional/screenshots @elastic/appex-sharedux
```

Comment:
I am not sure if this is us. Seems like there are mostly some screenshots of dashobards/visualizations

not sure how those work and what they are used for